capplets/windows/mate-window-properties.c

```diff
@@ -124,11 +122,8 @@ update_sensitivity (void)
 
     gtk_widget_set_sensitive (compositing_fast_alt_tab_checkbutton,
                               g_settings_get_boolean (marco_settings, MARCO_COMPOSITING_MANAGER_KEY));
-    gtk_widget_set_sensitive (enable_tiling_checkbutton,
-                              !g_settings_get_boolean (marco_settings, MARCO_REDUCED_RESOURCES_KEY));
```

Comment:
Why did you remove this when this preference is currently in use in marco?
```C
          !meta_prefs_get_reduced_resources ())
```


https://github.com/mate-desktop/marco/blob/aebd99901c99c49944bc514020f4517c545111c7/src/core/window.c#L1503

Reply:
PR #646

Reply:
see https://github.com/mate-desktop/mate-control-center/pull/641#issuecomment-821887597

The setting can still be enabled via dconf-editor or command line. I would vote for not expose a broken feature in mcc.

Also see

> Wireframe mode has not been updated/fixed to work with CSD windows
> and/or to exclude shadow/resize area from SSD windows. Also quickly
> testing with and without compositor wireframe sometimes leaves
> parts of frame on screen.
> 
> Considering that there has not been any bug report about broken
> wireframe mode I think it is time to remove it instead of wasting
> time trying to fix something that nobody is using.

(https://gitlab.gnome.org/GNOME/metacity/-/commit/aab0e5f7c390f2ba5c95498e5d28a88467b7f5f1)

Reply:
ok, don't worry